Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1161,6 +1171,9 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, two_be
             tests.add('TestCommonPython')  # test with no integration configured
             tests.add('HelloWorld-Test')  # test with integration configured
             packs_to_install.add("HelloWorld")
+    else:
```

Comment:
I think this should be in the end of `get_test_list_and_content_packs_to_install` along with the call to `get_packs_of_tested_integrations`